File: web/src/model/victron_controller/dashboard/from_0_2_0_wsservermessage_hello.ts
// @ts-nocheck
import {Hello as v0_2_0_Hello} from './v0_2_0/WsServerMessage'
import {Hello as dashboard_Hello} from './WsServerMessage'

export function convert__ws_server_message__hello__from__0_2_0(from: v0_2_0_Hello): dashboard_Hello {
    return new dashboard_Hello (
        from.server_version,
        from.server_ts_ms
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_wsservermessage_hello.rs


pub fn convert__wsservermessage__hello__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::ws_server_message::Hello) -> crate::victron_controller::dashboard::ws_server_message::Hello {
    crate::victron_controller::dashboard::ws_server_message::Hello {
        server_version: from.server_version.clone(),
        server_ts_ms: from.server_ts_ms.clone(),
        server_git_sha: None,
    }
}